src/core_plugins/metrics/public/components/vis_types/table/config.js

```diff
@@ -0,0 +1,113 @@
+import FieldSelect from '../../aggs/field_select';
+import Select from 'react-select';
+import YesNo from '../../yes_no';
+import ColorRules from '../../color_rules';
+
+class TableSeriesConfig extends Component {
+
+  componentWillMount() {
+    const { model } = this.props;
+    if (!model.color_rules || (model.color_rules && model.color_rules.length === 0)) {
+      this.props.onChange({
+        color_rules: [{ id: uuid.v1() }]
+      });
+    }
+  }
+
+  render() {
+    const defaults = { offset_time: '', value_template: '' };
+    const model = { ...defaults, ...this.props.model };
+    const handleSelectChange = createSelectHandler(this.props.onChange);
+    const handleTextChange = createTextHandler(this.props.onChange);
+
+    const functionOptions = [
+      { label: 'Sum', value: 'sum' },
+      { label: 'Max', value: 'max' },
+      { label: 'Min', value: 'min' },
+      { label: 'Avg', value: 'mean' },
+      { label: 'Overall Sum', value: 'overall_sum' },
+      { label: 'Overall Max', value: 'overall_max' },
+      { label: 'Overall Min', value: 'overall_min' },
+      { label: 'Overall Avg', value: 'overall_avg' },
+      { label: 'Cumlative Sum', value: 'cumlative_sum' },
+    ];
+
+    return (
+      <div>
+        <div className="vis_editor__series_config-container">
+          <div className="vis_editor__series_config-row">
+            <DataFormatPicker
+              onChange={handleSelectChange('formatter')}
+              value={model.formatter}
+            />
+            <div className="vis_editor__label">Template (eg.<code>{'{{value}}/s'}</code>)</div>
+            <input
+              className="vis_editor__input-grows"
+              onChange={handleTextChange('value_template')}
+              value={model.value_template}
+            />
+          </div>
+          <div className="vis_editor__series_config-row">
+            <div className="vis_editor__label">Filter</div>
+            <input
+              className="vis_editor__input-grows"
+              onChange={handleTextChange('filter')}
+              value={model.filter}
+            />
+            <div className="vis_editor__label">Show Trend Arrows</div>
+            <YesNo
+              value={model.trend_arrows}
+              name="trend_arrows"
+              onChange={this.props.onChange}
+            />
+          </div>
+          <div className="vis_editor__series_config-row">
+            <div className="vis_editor__label">Aggregate Column By</div>
+            <div className="vis_editor__row_item">
+              <FieldSelect
+                fields={this.props.fields}
+                indexPattern={this.props.panel.index_pattern}
+                value={model.aggregate_by}
+                onChange={handleSelectChange('aggregate_by')}
+              />
+            </div>
+            <div className="vis_editor__label">Aggregate Function</div>
+            <div className="vis_editor__row_item">
+              <Select
```

Comment:
This element is missing a [proper label](https://github.com/elastic/kibana/blob/master/style_guides/accessibility_guide.md#label-every-form-element).

You can add an `id` to the actual input field in `react-select` using:

```js
inputProps={{ id: 'someid' }}
```